packages/scandipwa/src/component/Field/Field.container.js

```diff
@@ -139,7 +139,8 @@ export class FieldContainer extends PureComponent {
             ? !!this.fieldRef.checked
             : this.fieldRef.value;
         const newValidRule = this.handleShowLengthError();
-        const response = validate(type === FIELD_TYPE.file
+        // eslint-disable-next-line max-len
+        const response = data === 'errorRequired' ? validate('', { isRequired: true }) : validate(type === FIELD_TYPE.file
```

Comment:
Please don't disable the eslint without a reason.
Split the line in this case.

Reply:
Actually, this line looks too complex.
Please refactor it. Maybe move it to the separate method and use the conditional block there.